indico/migrations/versions/20190925_1548_229bb73a558a_add_signature_token_column_to_user.py

```diff
@@ -0,0 +1,29 @@
+"""add signature token column to User
+
+Revision ID: 229bb73a558a
+Revises: f6fba869a27c
+Create Date: 2019-09-17 15:48:54.388997
+"""
+
+from alembic import op
+
+
+# revision identifiers, used by Alembic.
+revision = '229bb73a558a'
+down_revision = 'f6fba869a27c'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # give every user a 'signing_secret' column with a random UUID
+    # thanks to the author of https://stackoverflow.com/a/21327318
+    op.execute('''
+        ALTER TABLE users.users ADD COLUMN signing_secret uuid unique not null default
```

Comment:
I think you can keep using `op.add_column` and just put the default string inside `server_default` (probably as `sa.text()` or something similar that marks it as a literal sql string)

Reply:
and of course remove the `server_default` afterwards